src/panels/lovelace/views/hui-view.ts

```diff
@@ -136,6 +136,8 @@ export class HUIView extends LitElement {
           flex: 1 0 0;
           max-width: 500px;
           min-width: 0;
+          /* on iOS devices the column can become wider when toggling a switch */
```

Comment:
I wonder if we should fix this at the place where we render the switch ?

Reply:
I guess it can't hurt to do it here in HUI-view 🤷‍♂  We really don't want things to escape horizontally anyway.